### DISCUSSION TITLE
Welcome to romcal Discussions!

### DISCUSSION THREAD

**Discussion by emagnier** [romcal/romcal#302] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by emagnier** [comment]
I've enabled this new GitHub Discussions, to have a right place for any questions, ask for help, discussions, ideas (like the welcome message is saying just above), and to try to keep a cleanest issue tracker, with concise content as much as possible. GitHub Discussions is providing better tooling for all of that, instead of the issue tracker.

And if needed, it will be possible to convert an issue to a discussion, or create an issue from a discussion.

And precisely, I am thinking of moving some issues here in this GitHub Discussion, for example https://github.com/romcal/romcal/issues/208, so we can keep them better ordered or prioritized, and then create issues when things are clear of how it should be implemented in practice.

Hope you find that it makes sense 😉

**Reply by tukusejssirs** [reply]
Well, while I agree that question and help requests should be in GitHub Discussions, I don’t necessary agree on moving issues like #208 or other feature requests.

I suggest to keep the following in the GitHub Issues:
- bug reports;
- feature requests.

I understand you want to keep the issue tracker clean and concise though.

**Reply by emagnier** [reply]
#208 was just an example.

I want to move these issues here in GitHub Discussions, under the _Ideas_ or _General_ categories, because as we can see there are plenty discussions or exchanges on these topics, that means the subject could be too large, or not enough mature / well understood by everyone, or it could be not clear yet on how this could be integrated into romcal (use cases, goals, how to implement it...). Which in no way detracts from the original idea(s) of course!

GitHub Discussions will help to highlight the content in these discussions, and then create clean and precise feature request, issues....

As a developer, I can say it's not very appealing to open big issues like that, and try to synthesize and architecture the proposed feature request or enhancement.

**Reply by tukusejssirs** [reply]
Okay, do as you wish, just please: write some rules we could abide, like what and where should be placed in order to let me and others follow them.

**Comment by emagnier** [comment]
**Should I create a new Issue or a new Discussion?**

I think it's pretty straightforward. 

Create an **Issue**:
- For explicit topics (like decree that introduce new memorial, rank, date change, whatever) and simple feature request, bugs, enhancement, let’s continue as we’re doing.

Create a **Discussion**:
- For any questions, ask for help, want to explore an idea...
- For major new features, or anything that might require discussions to refine the scope, the user stories, the way to implement it... it's also better to start a new GitHub **Discussion**, especially that GitHub is providing great tools to better manage/follow/refine new ideas and features requests from a discussion. From that discussion we will be able then to create issue(s) when we are ready to implement it, or at least we all agree how it should be integrated into romcal.

In all cases, if you have any doubts about creating an **Issue** or a **Discussion**, create a **Discussion**.

Anyway, it's always possible to convert an issue to a discussion, or create a new issue from a discussion, so no stress 😄. Let's try it that way, and we will see if anything needs more clarification in the future (we are not the first GitHub repo using GitHub discussions, so I'm confident).

**Reply by tukusejssirs** [reply]
Thanks! Could we at least pin this comment/discussion or even better: include it into `README.md`?

**Reply by emagnier** [reply]
I've created a new pined discussion: https://github.com/romcal/romcal/discussions/304, including my previous message. 

I think it will have a better visibility like that. And if that is necessary in the future, I will edit this first message from this pined discussion. But we could also consider updating the CONTRIBUTING.md as well.
